WalletWasabi/Stores/IndexStore.cs

```diff
@@ -101,20 +135,31 @@ private async Task InitializeFiltersAsync()
 
 							lineTask = sr.EndOfStream ? null : sr.ReadLineAsync();
 
-							ProcessLine(height, line, enqueue: false);
-							height++;
+							ProcessLine(line, enqueue: false);
 
 							line = null;
 						}
 					}
 				}
+			}
+			catch
```

Comment:
Here i prefer `catch(Exception)` too.

Reply:
`catch(Exception)` is stupid. Removed it everywhere in the code.